package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -27,6 +27,51 @@
 
 from MDAnalysis.core.AtomGroup import AtomGroup
 
+class CoordinateNeighborSearch(object):
```

Comment:
I thought this had been removed a while ago by @kain88-de  in  #383 or thereabout.

Or did you add it??